ods_ci/tests/Tests/0500__ide/0501__ide_jupyterhub/custom-image.robot

```diff
@@ -94,9 +93,11 @@ Test Bad Image URL
 
 Test Image From Local registry
     [Documentation]  Try creating a custom image using a local registry URL (i.e. OOTB image)
-    ...       Issue reported for this test in the past - https://github.com/opendatahub-io/odh-dashboard/issues/2185
     [Tags]    Tier1
     ...       ODS-2470
+    ...       ExcludeOnDisconnected    # Since we don't have internal image registry enabled there usually
```

Comment:
Hi @jstourac this is the matter of an option in our automation where the internal registry is disabled. Moreover, the image registry can be enabled/disabled outside of disconnected automation (using `oc` command).

Reply:
I know, Andrej. But the test isn't passing now anyway. Also, since the linked issue won't be fixed probably, the test will probably go away at all.